ui/src/components/Toolbox.tsx

```diff
@@ -0,0 +1,75 @@
+import { MouseEventHandler, ReactNode, useState } from 'react';
+import Return from '@icons/Return.svg?react';
+import Rubber from '@icons/Rubber.svg?react';
+import Pencil from '@icons/Pencil.svg?react';
+import Palette from '@icons/Palette.svg?react';
+import Download from '@icons/Download.svg?react';
+
+// TODO: add event emiter when choose color (state) for each elements and onClick other elements (react-canvas interaction)
+// TODO: integrate with react-sketch-canvas (emiters?)
+
+interface Props {
+  icon: ReactNode;
+  text: string;
+  active: boolean;
+  onClick?: MouseEventHandler<HTMLDivElement>;
+}
+
+const ToolboxItem = ({ icon, text, active, onClick }: Props) => {
+  const style: string =
+    'flex cursor-pointer flex-col items-center justify-center gap-2 border-black p-1 text-center select-none hover:text-primary hover:fill-primary';
+
+  return (
+    <div className={active ? style + ' border font-bold' : style} onClick={onClick}>
```

Comment:
można by zainstlować bilbiotekę classnames https://www.npmjs.com/package/classnames i wtedy trochę uprościć ten zapis. 
```suggestion
    <div className={classNames(style, active && ' border font-bold')} onClick={onClick}>
```